CONTRIBUTING.md

```diff
@@ -48,11 +48,11 @@ Currently, these are the combinations of system dependencies that work on an int
 
 ```bash
 node -v
-v14.17.3
```

Comment:
Could you please see [this section](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md#nodejs-npm-pnpm) - and add similar instructions for this repo as well please?